.golangci.yml

```diff
@@ -48,13 +37,93 @@ linters:
   # - gochecknoglobals
   # - gochecknoinits
   # - gocyclo
-  # - gosec
   # - interfacer
   # - lll
   # - maligned
   # - prealloc
   # - stylecheck
   disable:
-  - errcheck
   - staticcheck
   - structcheck
+
+issues:
+  exclude-use-default: false
+  exclude-rules:
+  # Ignore errors when performing the following file operations. If these are
+  # not handled separately already, they tend to be insignificant.
+  - linters:
+    - errcheck
+    text: Error return value of `((body|buf|conn|file|io|ln|out|output|stdout|w|ws|watcher)\.(Copy|Flush|Write|WriteTo))` is not checked
```

Comment:
trying to enumerate all variable names that might store a writer is probably a losing battle.  can we just glob out the variable name here?

Reply:
The idea here was to err on the side of being more explicit. To ignore the error of one of these functions required looking at each use case and deciding it was the right thing to do.

I realize that this misses the places where—for example—`body` is used but the lint started passing because it was skipped in some other location.

So I can change this to a general glob, but just that was my reasoning for not using it here.

That being said, I do prefer to keep these two rules separate—the one that ignores `Copy`/`Flush`/etc and one that ignores `Close` as they feel separate enough to have a separate comment for each. I originally had these combined in a single regex, but again it felt better to just separate them; having a comment explaining each rather than a larger regex that is harder to understand if you're looking at it for the first time.

Reply:
Rather than listing all of the variable names here, should we just `//nolint` them at the sites?

Reply:
Yea at that point I think we're better using this exclude-rule and changing it to a glob if you'd rather not have them listed out. It results in a lot of `//nolint`s everywhere and there are very few places where we don't ignore these errors.

Reply:
If we think it's valuable to look at each case and decide ignoring the error is okay, probably better to `//nolint` each site.  On the other hand, if we're blanket okay with ignoring these errors in general, we can glob out the variable name.  I don't think the middle-ground of listing the variables here is the right approach.

Reply:
Sounds good. I'll change the regexes to use the glob regex but still keep the two rules separate just for clarify.